tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+	"compilerOptions": {
+		"module": "commonjs",
+		"declaration": true,
+		"noImplicitAny": true,
+		"removeComments": true,
+		"preserveConstEnums": true,
+		"strictNullChecks": true,
+		"sourceMap": true,
+		"target": "es5",
+		"noUnusedParameters": true,
+		"noUnusedLocals": true
```

Comment:
It does, and that's part of the `gulpfile`. You'll notice that the `.vscode/tasks.json` species that the build occurs via gulp, hence this is taken care of. You can build either via the commandline `gulp build`, or via VS Code. Other editors will need to be set up as required.